### PR TITLE
Add example, deprecated, default and positions for swagger 2.0

### PR DESCRIPTION
**Pull request by xclebel**

### CODE REVIEW

File: swagger-ext/src/test/scala/org/scalatra/swagger/SwaggerCommandSupportSpec.scala [hunk 1]
```diff
@@ -38,7 +38,6 @@ object SwaggerCommandSupportSpec {
     get("/new", endpoint("new"), operation(newOperation)) { "OK" }
 
     protected def applicationDescription: String = "The command support servlet"
-
```
Comment (malaporte) [inline swagger-ext/src/test/scala/org/scalatra/swagger/SwaggerCommandSupportSpec.scala:41]: gratuitous change 😛

File: swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala [hunk 2]
```diff
@@ -397,6 +397,7 @@ class SwaggerSpec2 extends ScalatraSpec with JsonMatchers {
             (JString(operationId) must_== operation \ "operationId") and
               verifyOperation(operation, swaggerJsonJValue \ "paths" \ path \ method, operationId)
         }
+      case _ => None
```
Comment (malaporte) [inline swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala:400]: Not sure I understand this one, looking at the data above it should never be neered, right...?

**Review by malaporte — APPROVED** [review]
Looks legits except a few questions.

File: swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala (hunk 2 above)
Comment (xclebel) [reply swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala:400]: The compiler was issuing a warning about it. Although I have to keep in mind this is not our code base and I'll keep the changes to the minimum

**Comment by xclebel** [comment]
This PR doesn't currently respect the Swagger spec 2.0, I'll have it modified

**Pull request closed without merging**